com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -396,115 +397,131 @@ private void UpdateStackCamerasOutput()
             Undo.RecordObject(camera, Styles.inspectorOverlayCameraText);
 
+            var serializedCameraSettings = serializedCamera.baseCameraSettings;
+
             bool isChanged = false;
 
             // Force same render texture
             RenderTexture targetTexture = settings.targetTexture.objectReferenceValue as RenderTexture;
-            if (camera.targetTexture != targetTexture)
+            if (cam.targetTexture != targetTexture)
             {
-                camera.targetTexture = targetTexture;
+                cam.targetTexture = targetTexture;
                 isChanged = true;
             }
 
             // Force same hdr
             bool allowHDR = settings.HDR.boolValue;
-            if (camera.allowHDR != allowHDR)
+            if (cam.allowHDR != allowHDR)
             {
-                camera.allowHDR = allowHDR;
+                cam.allowHDR = allowHDR;
                 isChanged = true;
             }
 
             // Force same mssa
             bool allowMSSA = settings.allowMSAA.boolValue;
-            if (camera.allowMSAA != allowMSSA)
+            if (cam.allowMSAA != allowMSSA)
             {
-                camera.allowMSAA = allowMSSA;
+                cam.allowMSAA = allowMSSA;
                 isChanged = true;
             }
 
             // Force same viewport rect
             Rect rect = settings.normalizedViewPortRect.rectValue;
-            if (camera.rect != rect)
+            if (cam.rect != rect)
             {
-                camera.rect = settings.normalizedViewPortRect.rectValue;
+                cam.rect = settings.normalizedViewPortRect.rectValue;
                 isChanged = true;
             }
 
             // Force same dynamic resolution
             bool allowDynamicResolution = settings.allowDynamicResolution.boolValue;
-            if (camera.allowDynamicResolution != allowDynamicResolution)
+            if (serializedCamera.allowDynamicResolution.boolValue != allowDynamicResolution)
             {
-                camera.allowDynamicResolution = allowDynamicResolution;
+                cam.allowDynamicResolution = allowDynamicResolution;
                 isChanged = true;
             }
 
             // Force same target display
             int targetDisplay = settings.targetDisplay.intValue;
-            if (camera.targetDisplay != targetDisplay)
+            if (cam.targetDisplay != targetDisplay)
             {
-                camera.targetDisplay = targetDisplay;
+                cam.targetDisplay = targetDisplay;
                 isChanged = true;
             }
 
-            // Force same target display todo
-            StereoTargetEyeMask stereoTargetEye = (StereoTargetEyeMask)settings.targetEye.intValue;
-            if (camera.stereoTargetEye != stereoTargetEye)
+#if ENABLE_VR && ENABLE_XR_MODULE
+            // Force same target display
+            int selectedValue = !m_SerializedCamera.allowXRRendering.boolValue ? 0 : 1;
```

Comment:
shouldnt it be 
!boolValue? 0 : boolValue
?

Reply:
we are casting the  bool value in an inverse way. So it is correct. In the other usages is done in the same way